packages/blockchain-wallet-v4-frontend/src/modals/UpgradeAddressLabels/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+
+import modalEnhancer from 'providers/ModalEnhancer'
+import UpgradeAddressLabels from './template.js'
+
+class UpgradeAddressLabelsContainer extends React.PureComponent {
+  render () {
+    return <UpgradeAddressLabels {...this.props} />
+  }
+}
+
+UpgradeAddressLabelsContainer.propTypes = {
+  duration: PropTypes.number
+}
+
+const enhance = compose(
+  modalEnhancer('UpgradeAddressLabels'),
+  connect(undefined, undefined)
+)
+
+export default enhance(UpgradeAddressLabelsContainer)
```

Comment:
Can just be `export default modalEnhancer('UpgradeAddressLabels')(UpgradeAddressLabelsContainer)` since you don't need to `connect` to redux.